### ISSUE TITLE
voc2007 test 

### ISSUE THREAD

**Issue by hsl8856** [mohhao/TF-Keras-ThunderNet#4]
The test results are all background questions.,. Have you solved it? 
What is the final convergence result of your training loss function(RPN,CLass)? thanks

**Comment by ABCAI93** [comment]
Hello, thanks for your sharing. But I encounted the same problem, have you solved it?
Besides, the speed of my training seems to be a little slow, may I ask you which GPU you use and how long it will take for training one epoch?

**Comment by hsl8856** [comment]
Sorry, the previous question has not been solved. I think it is caused by missing ＂bg＂ items in the training data set.




At 2019-11-04 14:57:43, "YONGHUICAI" <notifications@github.com> wrote:


Hello, thanks for your sharing. But I encounted the same problem, have you solved it?
Besides, the speed of my training seems to be a little slow, may I ask you which GPU you use and how long it will take for training one epoch?
